website/pages/blog/auditing-service-account-keys-with-cloudquery.md

```diff
@@ -0,0 +1,180 @@
+somewhere your systems can access. This could be a file,
+environment, variable, or bucket.
+
+Unfortunately, with this highly flexible global bearer strategy,
+tracking the location of the key becomes unmanageable. Multiple 
+users and services can use the same key, defeating the 
+non-repudiation principle. It also creates perverse incentives,
+where reusing an existing key becomes easier, than creating a new,
+purpose built key.
+
+This makes understanding your system and access policy difficult,
+and amplifies the impact of lost or stolen keys. Additionally,
+rotating these keys because a complex and difficult challenge,
+as it's difficult to find every use of the key, and even more difficult
+to find a window where all services and uses can be upgraded together.
+
+As the needs of your services evolve, these Service Accounts tend
+to gain more permissions, which leaves overly permissed keys. All
+services should have their own accounts with the minimum permissions
+required.
+
+# Service Accounts vs Service Account Keys
+
+Service Accounts are accounts created for use by a service.
+These are generally not interactive, often programmatic accounts.
+Service Accounts allow you to associate an identity with a workload,
+and allow permissions to be attached to that identity.
+
+Service Account Keys, are cryptographic keys or shared secrets that
+act as a form a proof to authenticate a call to a Service Account.
+Service Account Keys are just another way to use Service Accounts, albeit
+one with significant risks.
+
+
+Typically, the account keys would look something like this:
+
+GCP Service Account Key:
+```yaml
+id: "key ID"
+private_key: >-
+  -----BEGIN PRIVATE KEY-----
+  PRIVATE_KEY
+  -----END PRIVATE KEY-----
+client_id: "client ID"
+...
+```
+
+AWS Account Key:
+```yaml
+AccessKeyID: "AKIAIOSFODNN7EXAMPLE"
+SecretKey:   "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+```
+
+## Service Account Keys vs Secrets
+
+It is probably tempting to store your Service Account Keys in
+a Key or Secret Management system such as AWS Secret Manager, GCP Secrets Manager,
+Azure KeyVault, or Hashicorp Vault. These systems allow secure
+dissemination of secret material to services or users.
+
+However, this approach is also fraught with peril.
+
+Although secret managers allow secure storage, once the secret material leaves
+the storage system, the security of the secret material is reliant on the client,
+and is vulnerable to leaks.
+Additionally, accessing this secret material requires authentication and authorization,
+implying some preexisting privilege and identity. Regardless of the secure storage,
+the identity problem still persists.
+
+Since you are already using Identity and Access Management to access your secrets, 
+why not lean into the IAM platform and use it for authorizing the end service directly?
+
+## AWS
+
+AWS has a feature called [Access Keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html). 
```

Comment:
Were Access Keys referenced before in the `AWS Account Key` Section?  I may be missing some context here.  Would be good to clarify differences between these sections.